### DISCUSSION TITLE
Script cannot find resource group

### DISCUSSION THREAD

**Discussion by pererap01** [PrateekKumarSingh/AzViz#87] [General]
I am getting the following error when attempting to run the export wizard

![image](https://user-images.githubusercontent.com/24482168/158485046-d5d71517-85a6-4e07-b10b-6d47de5dd74a.png)

and this is a table that shows the rg exisits.
![image](https://user-images.githubusercontent.com/24482168/158484951-54f87877-dd8c-4f66-bbcb-5e582243274a.png)

any ideas why this is happening

**Comment by PrateekKumarSingh** [comment]
@pererap01 make sure you are setting the correct Azure Context before running the `Export-AzViz` cmdlet, using `Set-AzContext -Subscription <subcriptionId-where-your-resource-group-exists> `

please let me know if the issue still persists, thank you
